changelog.d/15668.doc

```diff
@@ -0,0 +1 @@
+Documentation improvements to contributing guide. Contributed by @grantm
```

Comment:
The CI is complaining because this needs some punctuation at the end

```suggestion
Documentation improvements to contributing guide. Contributed by @grantm.
```

We could improve it slightly as well:


```suggestion
Improve contributor docs to make it more clear that Rust is a necessary prerequisite. Contributed by @grantm.
```